cmd/codegen/hack/patch_crd_descriptions.sh

```diff
@@ -18,7 +18,7 @@ run_yq() {
     image="fleet-codegen-hack:yq"
     log "yq (from https://github.com/kislyuk/yq) is missing, building a helper docker image ($image)..."
 
-    docker build -t $image - << EOF
+    docker build -t $image - << EOF >/dev/null
```

Comment:
May I suggest redirecting to stderr instead?
```suggestion
    docker build -t $image - >&2 << EOF
```